src/interactivity/base/ServiceLocator.cpp

```diff
@@ -104,6 +104,26 @@ void ServiceLocator::RundownAndExit(const HRESULT hr)
 
 #pragma region Set Methods
 
+[[nodiscard]] NTSTATUS ServiceLocator::SetConsoleControlInstance(_In_ std::unique_ptr<IConsoleControl>&& control)
+{
+    NTSTATUS status = STATUS_SUCCESS;
+
+    if (s_consoleControl)
+    {
+        status = STATUS_INVALID_HANDLE;
+    }
+    else if (!control)
+    {
+        status = STATUS_INVALID_PARAMETER;

```

Comment:
I wish these were return statements.

Reply:
Wish granted. I was just matching the style of the file but sure.